repository-data/webfiles/src/main/resources/site/freemarker/common/macro/fileInfographic.ftl

```diff
@@ -0,0 +1,10 @@
+<#ftl output_format="HTML">
+<#include "../../include/imports.ftl">
+<#include "component/infoGraphic.ftl">
+
+<#macro fileInfographic mimeType = "">
```

Comment:
Is this file used anywhere?

Reply:
No idea. It could be a question for Gareth.